### ISSUE TITLE
Feature: Cassandra Query Language (cql) support

### ISSUE THREAD

**Issue by peter-janssen** [google/code-prettify#451]
What do I need to do to get cql supported in code-prettify? Do I need to supply a regex in a new lang-cql.js file or is there more to it?

**Comment by amroamroamro** [comment]
Yes, you must create a new extension `lang-cql.js` for that language. This consists of regex patterns to detect the different constructs (comments, strings, keywords, etc..). Finally there's a call to PR.registerLangHandler Register the language handler for the given file extensions.

Take a look at the existing extensions in `src/lang-*.js` to get an idea.
